.circleci/config.yml

```diff
@@ -1,51 +1,26 @@
----
+version: 2.1
+
 release_tags: &release_tags
   tags:
     only: /^v\d+(\.\d+){2}(-.*)?$/
 
-test_env: &test_env
-  OPENCENSUS_MONGODB_TESTS: 1
-  OPENCENSUS_REDIS_TESTS: 1
-
-# "Include" for unit tests definition.
-unit_tests: &unit_tests
-  steps:
-    - checkout
-    - run:
-        name: Install modules and dependencies.
-        command: npm install
-    - run:
-        name: Check code style and linting
-        command: npm run check
-    - run:
-        name: Ensure code compiles to JS.
-        command: npm run compile
-    - run:
-        name: Run unit tests.
-        command: npm run test
-    - run:
-        name: Submit coverage data to codecov.
-        command: npm run codecov
-        when: always
-
-mongo_service: &mongo_service
-  image: mongo
-redis_service: &redis_service
-  image: redis
-
-version: 2.0
 workflows:
   version: 2
   tests:
     jobs:
-      - node8:
-          filters: *release_tags
-      - node10:
-          filters: *release_tags
-      - node11:
+      # Separate node8 from others to ignore any packages that no longer
+      # support node8
+      - unit_tests:
+          name: node8
           filters: *release_tags
-      - node12:
+          node_version: 8
+          npm_script_extra_args: --ignore @opencensus/exporter-stackdriver
+      - unit_tests:
+          name: node<< matrix.node_version >>
           filters: *release_tags
+          matrix:
+            parameters:
+              node_version: [10, 11, 12]
```

Comment:
Not related to this PR, but in future we should consider not adding support for new unstable (odd-numbered) versions of NodeJS.

Reply:
agreed

Reply:
Should we just leave these versions and not add any?